helper/resource/importstate/import_block_with_id_test.go

```diff
@@ -288,6 +290,12 @@ func TestImportBlock_WithID_WithBlankOptionalAttribute_GeneratesCorrectPlan(t *t
 
 				resource "examplecloud_container" "test" {
 					name = "somename"
+				}
+
+				import {
```

Comment:
This test change is a consequence of #484.

Curiously, the tests did not run in https://github.com/hashicorp/terraform-plugin-testing/pull/484 💭